vector/v.cluster/main.c

```diff
@@ -264,7 +264,7 @@ int main(int argc, char *argv[])
 	    c[2] = 0.0;
 	    n = 0;
 	    sum = sumsq = 0;
-	    min = 1.0 / 0.0;
+		sscanf("inf", "%lf", &min);
```

Comment:
In #2681, the code was changed to use `INFINITY`, but there's a note that it might not for MSVC (at that time)

```c
            min = INFINITY;
```